sqs/Module.md

```diff
@@ -1,83 +1,25 @@
-[![Build Status](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs.svg?branch=master)](https://travis-ci.org/ballerina-platform/module-ballerinax-aws.sqs)
+## Overview
+Ballerina connector for Amazon SQS is connecting the Amazon SQS API via Ballerina language easily. It provides capability to perform operations related to queues and messages.
 
-# Ballerina Amazon SQS Connector
+This module supports [Amazon SQS API](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/welcome.html) version 2012-11-05.
 
-Amazon SQS Connector allows you to connect to the Amazon Simple Queue Service (SQS) via REST API from Ballerina.
+## Prerequisites
+Before using this connector in your Ballerina application, complete the following:
+* Create [AWS account](https://aws.amazon.com)
+* Obtaining tokens
+        
+    Follow [this link](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/) and obtain the Access key ID, Secret access key, Region and Account number from navigation pane `Users` and selecting `Security credentials` tab.
```

Comment:
Can we convert this to some clear steps? Check the following:

1 [Create and activate your AWS account](https://aws.amazon.com/premiumsupport/knowledge-center/create-and-activate-aws-account/).
2. To obtain the access key ID, secret access key, region and account number, click **Users** on the navigation pane and then click **Security Credentials**."

Is substep 1 the same as step 1? If so, have only the second step as text under "Obtain Tokens".

Reply:
Fixed by adding as one line